src/renderer/entities/wallet/lib/permission-utils.ts

```diff
@@ -0,0 +1,143 @@
+import { Account, ProxiedAccount, Wallet } from '@shared/core';
+import { walletUtils } from './wallet-utils';
+
+export const permissionUtils = {
+  isTransferAvailable,
+  isReceiveAvailable,
+  isStakingAvailable,
+  isCreateMultisigTxAvailable,
+  isApproveMultisigTxAvailable,
+  isRejectMultisigTxAvailable,
+  isCreateAnyProxyAvailable,
+  isCreateNonAnyProxyAvailable,
+  isRemoveProxyAvailable,
+};
```

Comment:
With suggested naming we're closer to actions (because of verb `can`)
`is` is generally used for property or feature of the object 
As instance - `canTransfer` tells us about action, `isTransferAvailable` is more of a characteristic
```suggestion
export const permissionUtils = {
  canTransfer,
  canReceive,
  canStake,
  canCreateMultisig,
  canApproveMultisig,
  canRejectMultisig,
  canCreateAnyProxy,
  canCreateNonAnyProxy,
  canRemoveProxy,
};
```